### PR TITLE
Enhancement: Suppor for --ignore-platform-reqs

### LINKED ISSUE #9: php extensions
I started playing with this action today and pretty quickly run into a 'dead end' as far as php extension based sources is concerned.

the check errors because the php-docker container does not contain the required php extensions (or at least they are not loaded/configured):

image

would it be possible to e.g. use a docker container which contains the most used/popular php extensions (or maybe those which are bundled with ubuntu by default or similar)

### PR DESCRIPTION
**Pull request by OskarStark**
Closes #9 

Pleas review @greg0ire @Nyholm @sagikazarmark @staabm

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,6 +55,24 @@ jobs:
...
+jobs:
+  phpstan:
+    name: PHPStan
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: PHPStan
+      uses: docker://oskarstark/phpstan-ga
++     env:
++        CHECK_PLATFORM_REQUIREMENTS: false
```
Comment (Nyholm) [inline README.md:71]: I suggest to reverse the logic. 

Name it `IGNORE_PLATFORM_REQUIREMENTS` instead. That will be the same as the composer flag

**Review by Nyholm — COMMENTED** [review]
Good. I just have one suggestion

File: README.md (hunk 1 above)
Comment (OskarStark) [reply README.md:71]: I know, but it depends on the default, the default for `NO_DEV` is false, so you set it to true. The default, in this case, is to *not* ignore platform reqs which sounds weird, that `IGNORE_PLATFORM_REQS: false` will actually ignore the platform reqs.
I did this in the first try.

Comment (Nyholm) [reply README.md:71]: Hm.. Yeah. Okey. 

Yeah, that is fair. I didnt consider the other flag.

**Review by Nyholm — APPROVED** [review]

**Comment by staabm** [comment]
thanks for the ping,... this looks good to me.

for the few months since I opened the mentioned issue I learned for myself, that setting up a plain php container and installing everything by hand is most flexbibel (and not even that much slower/more time intensive, then using a action such like this phpstan-ga one).

use `shivammathur/setup-php@v2` as a base and install phpstan via composer (like we did with travis in the past) and you are done.

File: README.md (hunk 1 above)
Comment (greg0ire) [reply README.md:71]: Fully agree with @OskarStark here, I always try to avoid flags with "negative ideas", such as `DISABLE_XXX`.

**Review by greg0ire — APPROVED** [review]

**Comment by OskarStark** [comment]
Thanks, I created a release

**Comment by greg0ire** [comment]
cc @franmomu :)

**Pull request merged**